src/midi/midi_fluidsynth.cpp

```diff
@@ -98,16 +133,36 @@ bool MidiHandlerFluidsynth::Open(MAYBE_UNUSED const char *conf)
 		return false;
 	}
 
+	// Load the requested SoundFont or quit if none provided
 	std::string soundfont = section->Get_string("soundfont");
+	int scale_by_percent = split_postfixed_percent(soundfont, 100);
+
 	Cross::ResolveHomedir(soundfont);
 	if (!soundfont.empty() && fluid_synth_sfcount(fluid_synth.get()) == 0) {
 		fluid_synth_sfload(fluid_synth.get(), soundfont.data(), true);
 	}
-	DEBUG_LOG_MSG("MIDI: FluidSynth loaded %d SoundFont files",
-	              fluid_synth_sfcount(fluid_synth.get()));
+	if (fluid_synth_sfcount(fluid_synth.get()) == 0) {
+		LOG_MSG("MIDI: FluidSynth failed to load %s, check the path.",
+		        soundfont.c_str());
+		return false;
+	}
 
-	// Uses samples' native amplitudes without suppression or amplification
-	fluid_synth_set_gain(fluid_synth.get(), 1.0);
+	if (scale_by_percent < 1 || scale_by_percent > 500) {
+		LOG_MSG("MIDI: FluidSynth invalid scaling of %d%% provided; resetting to 100%%",
+		        scale_by_percent);
+		scale_by_percent = 100;
+	}
+	fluid_synth_set_gain(fluid_synth.get(),
+	                     static_cast<float>(scale_by_percent) / 100.0f);
+
+	// Let the user know that the SoundFont was loaded
+	if (scale_by_percent == 100)
+		LOG_MSG("MIDI: FluidSynth loaded %s.", soundfont.c_str());
+	else
+		LOG_MSG("MIDI: FluidSynth loaded %s with levels %s by %d%%.",
+		        soundfont.c_str(),
+		        scale_by_percent > 100 ? "amplified" : "attenuated",
```

Comment:
Just a note, as this is something I learned very recently.

If we want to have translatable strings in the future, we should refrain from injecting words via %s; that's because translation in different language might require putting the word in a different position and translators usually can't correlate the context of different words the way us, programmers can.

In this case it doesn't really matter, as this is a log msg (and I don't know if we want logs translated?); if these were strings to be displayed e.g. via DOS shell, then translation-friendly version of this code would be:
```
	else if (scale_by_percent > 100)
		LOG_MSG("MIDI: FluidSynth loaded %s with levels amplified by %d%%.",
		        soundfont.c_str(), scale_by_percent);
	else
		LOG_MSG("MIDI: FluidSynth loaded %s with levels attenuated by %d%%.",
		        soundfont.c_str(), scale_by_percent);
```